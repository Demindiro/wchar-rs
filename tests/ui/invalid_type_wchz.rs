use wchar::wchz;

const INVALID: &[f32] = wchz!(f32, "oops");

fn main() {}
