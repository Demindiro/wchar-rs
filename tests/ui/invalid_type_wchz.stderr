error: unexpected wchar type `f32`, expected one of: u16, u32, i16, i32
 --> $DIR/invalid_type_wchz.rs:3:31
  |
3 | const INVALID: &[f32] = wchz!(f32, "oops");
  |                               ^^^
